skimage/io/collection.py

```diff
@@ -126,81 +125,41 @@ class ImageCollection:
     Note that files are always returned in alphanumerical order. Also note
     that slicing returns a new ImageCollection, *not* a view into the data.
 
-    ImageCollection image loading can be customized through
-    `load_func`. For an ImageCollection ``ic``, ``ic[5]`` calls
-    ``load_func(load_pattern[5])`` to load that image.
-
-    For example, here is an ImageCollection that, for each video provided,
-    loads every second frame::
-
-      import imageio.v3 as iio3
```

Comment:
The new example added below is nice, but I think one of these video-reader examples should stay, since they reflect a very typical real-world application. The code here isn't run anyway, so it wouldn't hurt to keep it?

Reply:
Addressed in https://github.com/scikit-image/scikit-image/pull/6988/commits/0195fd825bf72f1337e377074e1b753e174b4b59. Let me know what you think. I quite like that we can now use the "Palisades of Vogt" dataset as a nice "movie-like" example. I also made sure to discuss the bottleneck that is re-opening the file with ImageCollection. :)